third-party-podspecs/Folly.podspec

```diff
@@ -10,17 +10,17 @@ Pod::Spec.new do |spec|
   spec.module_name = 'folly'
   spec.dependency 'boost-for-react-native'
   spec.dependency 'DoubleConversion'
-  spec.dependency 'GLog'
+  spec.dependency 'glog'
   spec.compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1'
-  spec.source_files = 'folly/Bits.cpp',
-                      'folly/Conv.cpp',
-                      'folly/Demangle.cpp',
-                      'folly/StringBase.cpp',
-                      'folly/Unicode.cpp',
-                      'folly/dynamic.cpp',
-                      'folly/json.cpp',
-                      'folly/portability/BitsFunctexcept.cpp',
-                      'folly/detail/MallocImpl.cpp'
+  spec.source_files = 'folly/Bits.{h,cpp}',
```

Comment:
Why do we add `.h` files here?

Reply:
We are creating static frameworks and using that instead of directly using pods. Headers are needed to create folly framework. Will revert it. For us would be good to have headers here. Please let me know if it will be merged, I will create a separate pull request with changed headers. Thanks.

Reply:
To clarify, we are doing that for build performance.